x/bank/client/cli/query.go

```diff
@@ -105,6 +105,7 @@ Example:
 func GetCmdDenomsMetadata() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "denom-metadata",
+		Args:  cobra.NoArgs,
```

Comment:
RunE function takes args as `_` (`RunE: func(cmd *cobra.Command, _ []string)`). 
Is it because `GetCmdDenomsMetadata` does not takes args?

Reply:
Yes, I think.

Reply:
Both of my comments are about inconsistency of code. 
Should these things be fixed in this PR? Or is it better to leave it as an issue in upstream?

Reply:
I think the inconsistency should be resolved by another PR. Because the upstream may reject the modification about it.